pipelines/upload_transacao_cct/tasks.py

```
@@ -0,0 +1,307 @@
+                    sql = f"""
+                        CREATE INDEX idx_tmp_id_transacao
+                        ON public.{tmp_table_name} (id_transacao)
+                    """
+                    log("Criando índice tabela temporária")
+                    cur.execute(sql)
+
+                    sql = f"""
+                        DELETE FROM public.{table_name} t
+                        USING public.{tmp_table_name} s
+                        WHERE t.id_transacao = s.id_transacao
+                    """
```

Ruff (0.14.0)

Possible SQL injection vector through string-based query construction

(S608)